### ISSUE TITLE
Add the test diagrams in the source tarball

### ISSUE THREAD

**Issue by tk0miya** [blockdiag/seqdiag#17]
Hello,

I'm currently packaging *diag for Fedora, and this package's test suite fails:

```
ERROR: Failure: OSError ([Errno 2] No such file or directory: '.../seqdiag-0.9.0/src/seqdiag/tests/diagrams/')
```

Running the test suite is not mandatory, but I'd rather run it and make sure nothing's broken on Fedora. It also helped me discover that blockdiag relies by default on `ipagp.ttf` for instance.

I'm picking source tarballs from:
https://pypi.python.org/pypi/seqdiag/

Thank you for *diag, I wish I was told about it sooner :)

----------------------------------------
- Bitbucket: https://bitbucket.org/blockdiag/seqdiag/issue/17
- Originally reported by: [dridi](https://bitbucket.org/dridi)
- Originally created at: 2013-12-23T07:50:35.020

**Comment by tk0miya** [comment]
_From [dridi](https://bitbucket.org/dridi) on 2014-03-02 21:37:05+00:00_

Hello,

My current workaround is to get the source tarball from bitbucket (same problem with nwdiag). The four *diag should land in a couple weeks in Fedora.

**Comment by tk0miya** [comment]
_From Takeshi KOMIYA on 2014-06-13 10:45:33+00:00_

Fixed at 14048a6e7e67 .
they will be included from next release.

Thanks.

**Comment by tk0miya** [comment]
_From [dridi](https://bitbucket.org/dridi) on 2014-06-13 13:18:44+00:00_

Thanks, I'm looking forward to the next release!
